### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by zstundys** [PLTytus/GTAWeb#92]
**Describe your idea:**
I find myself pressing "Esc" to close the fullscreen image preview -- failing to do so -- I can only close the fullscreen image preview by using a mouse. I think "Esc" to press top-right "X" would be a nice a11y feature.

**Comment by PLTytus** [comment]
Done
